pkg/controllers/migration/vmware.go

```diff
@@ -42,15 +42,18 @@ func (h *vmwareHandler) OnSourceChange(_ string, v *migration.VmwareSource) (*mi
 		"kind":      v.Kind,
 		"name":      v.Name,
 		"namespace": v.Namespace,
-	}).Info("Reconciling migration source")
+		"key":       key,
```

Comment:
we already have object Name and object Namespace in the fields. Key is just the namespaced name. Do we still need it?

Reply:
You are right. Will remove it.